### PR TITLE
Update build and integrate KVM

### PR DESCRIPTION
**Pull request by Toasterson**
These are my latest changes based from xen0l/oi-packer and currently the base of my Vagrantcloud boxes. The configuration got migrated to hcl2 which is new in packer  1.5 Additionally I integrated KVM support from PR #2 

Currently, the VirtualBox VM works and is available at: https://app.vagrantup.com/Toasterson/boxes/openindiana-hipster
The KVM build fails to bring up the NIC, due to the fact, that OpenIndiana identifies the NIC inside the Vagrant box as a different NIC to the one used during installation. Thus, no network configuration is present for the new NIC. I still need to find a solution for that issue but otherwise the box works.

### CODE REVIEW

**Comment by Toasterson** [comment]
Currently latest releases are available at https://app.vagrantup.com/Toasterson/boxes/openindiana-hipster/versions/2020.04.13 will run both builds again next month with current release.

File: README.md [hunk 1]
```diff
@@ -18,24 +18,39 @@ Please note, that box building works only on platforms where packer runs.
 * Download VirtualBox from the official [download site](https://www.virtualbox.org/wiki/Downloads). We recommend to  always use the latest VirtualBox.
 * Run the following commands:
 
-  ```
-  git clone https://github.com/OpenIndiana/oi-packer.git && \
-  cd oi-packer && \
-  packer build hipster.json
-  ```
+```bash
+git clone https://github.com/Toasterson/oi-packer.git && \
```
Comment (pyhalov) [inline README.md]: Likely this should refer OI repo

**Comment by pyhalov** [comment]
Can you rebase it on master to avoid conflicts?

File: README.md (hunk 1 above)
Comment (Toasterson) [reply README.md]: Yes it should and does now :)

**Comment by Toasterson** [comment]
Rebased as asked does this help?

**Comment by Toasterson** [comment]
KVM build is fixed BTW. both builds work and produce running boxes

**Comment by Toasterson** [comment]
Needed to remove an email that accidentally ended up in my git config.

**Pull request merged**